packages/hub/src/app/models/[owner]/[slug]/EditSquiggleSnippetModel.tsx

```diff
@@ -36,12 +39,66 @@ import {
   draftUtils,
   useDraftLocator,
 } from "./SquiggleSnippetDraftDialog";
+import { FormModal } from "@/components/ui/FormModal";
 
 export type SquiggleSnippetFormShape = {
   code: string;
   relativeValuesExports: RelativeValuesExportInput[];
 };
 
+type OnSubmit = (extraData?: { comment: string }) => Promise<void>;
+
+const SaveDialog: FC<{ onSubmit: OnSubmit; close: () => void }> = ({
+  onSubmit,
+  close,
+}) => {
+  type SaveFormShape = {
+    comment: string;
+  };
+  const form = useForm<SaveFormShape>(); // TODO
```

Comment:
If you leave a TODO, could be good if you give a bit of context.

Reply:
I forgot to remove the comment.